### DISCUSSION TITLE
Argument of type '"generate:done"' is not assignable to parameter of type 'HookKeys<NuxtHooks>'

### DISCUSSION THREAD

**Discussion by jsoneaday** [nuxt/nuxt#18634] [Questions]
I am trying to create a module using defineNuxtModule. Inside of the setup function I do the code below but it fails on the 'generate:done' parameter with the error given. Why does it not see this hook?

`async setup(options, nuxt) {
        nuxt.hook('generate:done', async (_) => {`

**Comment by jsoneaday** [comment]
Looks like generate:done is no longer an available hook

**Comment by Baroshem** [comment]
Hey, 

here is a list of available hooks -> https://nuxt.com/docs/api/advanced/hooks
